File: app/src/main/java/cn/intret/app/picgo/model/RemoveFileListMessage.java
package cn.intret.app.picgo.model;

import java.io.File;
import java.util.List;

class RemoveFileListMessage {
    List<File> mRemovedFiles;
    List<File> mUnRemovedFiles;
    File mDir;
    int mTotal;

    public List<File> getRemovedFiles() {
        return mRemovedFiles;
    }

    public RemoveFileListMessage setRemoveSuccessFiles(List<File> removedFiles) {
        mRemovedFiles = removedFiles;
        return this;
    }

    public List<File> getUnRemovedFiles() {
        return mUnRemovedFiles;
    }

    public RemoveFileListMessage setRemoveFailedFiles(List<File> unRemovedFiles) {
        mUnRemovedFiles = unRemovedFiles;
        return this;
    }

    public File getDir() {
        return mDir;
    }

    public RemoveFileListMessage setDir(File dir) {
        mDir = dir;
        return this;
    }

    public int getTotal() {
        return mTotal;
    }

    public RemoveFileListMessage setTotal(int total) {
        mTotal = total;
        return this;
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/event/RemoveFileListMessage.kt
package cn.intret.app.picgo.model.event

import java.io.File

internal class RemoveFileListMessage {
    lateinit var removedFiles: List<File>
    lateinit var unRemovedFiles: List<File>
    lateinit var mDir: File
    var mTotal: Int = 0

    fun setRemoveSuccessFiles(removedFiles: List<File>): RemoveFileListMessage {
        this.removedFiles = removedFiles
        return this
    }

    fun setRemoveFailedFiles(unRemovedFiles: List<File>): RemoveFileListMessage {
        this.unRemovedFiles = unRemovedFiles
        return this
    }

    fun getDir(): File {
        return mDir
    }

    fun setDir(dir: File): RemoveFileListMessage {
        mDir = dir
        return this
    }

    fun getTotal(): Int {
        return mTotal
    }

    fun setTotal(total: Int): RemoveFileListMessage {
        mTotal = total
        return this
    }
}
